Formula/podman.rb

```diff
@@ -46,7 +43,14 @@ def install
     ENV["CGO_ENABLED"] = "1"
     os = OS.kernel_name.downcase
 
-    ENV["HELPER_BINARIES_DIR"] = libexec
+    if OS.linux?
+      # Update helper binary location to look at libexec/podman inside the keg.
+      # This is not needed on Mac because config_darwin.go already includes this
+      # location, but Linux doesn't so let's add it through the environment
+      # variable. Note that we want a literal `$BINDIR`, so we need to escape
+      # the `$` for Makefile syntax, so we need to double it as `$$BINDIR`.
+      ENV["HELPER_BINARIES_DIR"] = "$$BINDIR/../libexec/podman"
+    end
```

Comment:
```suggestion
    # Update helper binary location to look at libexec/podman inside the keg.
    # This is not needed on Mac because config_darwin.go already includes this
    # location, but Linux doesn't so let's add it through the environment
    # variable. Note that we want a literal `$BINDIR`, so we need to escape
    # the `$` for Makefile syntax, so we need to double it as `$$BINDIR`.
    ENV["HELPER_BINARIES_DIR"] = "$$BINDIR/../libexec/podman" if OS.linux?
```

Reply:
Updated.